tests.c

```diff
@@ -158,6 +159,224 @@ test_opcode_0x13(void) // NOLINT
   CU_ASSERT(cpu.d == 1);
 }
 
+void
+test_opcode_0x05(void) // NOLINT
+{
+  i8080 cpu;
+  cpu_init(&cpu);
+
+  cpu.b = 0xF;                                      // NOLINT
+  int code_found = execute_instruction(&cpu, 0x05); // NOLINT
+
+  CU_ASSERT(code_found == 0);
+  CU_ASSERT_EQUAL(cpu.b, 14);
+  CU_ASSERT((cpu.flags & FLAG_Z) == 0);
+  CU_ASSERT((cpu.flags & FLAG_S) == 0);
+  CU_ASSERT((cpu.flags & FLAG_P) == 0);
+  CU_ASSERT((cpu.flags & FLAG_AC) == FLAG_AC);
+}
+
+void
+test_opcode_0x0e(void)
+{
+  // MVI C, D8
+  i8080 cpu;
+  cpu_init(&cpu);
+
+  cpu.pc = 0x4341;                       // NOLINT
+  cpu_write_mem(&cpu, cpu.pc + 1, 0x19); // NOLINT
```

Comment:
Should clean this up so the memory write doesn't persist to the next test.

cpu_write_mem(&cpu, 0x4342, 0x00);